### DISCUSSION TITLE
Have a release of GitHub Mobile with Intune SDK

### DISCUSSION THREAD

**Discussion by byjrack** [community/community#4071] [Mobile]
Should allow for associates to access orgs etc. where there are additional access controls like IP Address Allow Lists.

**Comment by aconsuegra** [comment]
Thank you so much for the request! We are tracking being able to manage the app with Intune internally and I added your note to our issue. In the meantime, please let us know if you have any other bugs or feature requests. We really appreciate it!

**Reply by byjrack** [reply]
With EMUs as GA any progress on an Intune enlightened version?  The oidc hopefully shouldn't be too much work, copy/paste controls, and maybe take advantage of azure app proxies so any flows come from known IP space.
